docs/reference/api/linea-estimategas.mdx

```diff
@@ -0,0 +1,71 @@
+---
+description: Reference content for the linea_estimateGas method.
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# `linea_estimateGas`
+
+Generates and returns an estimate of how much gas is necessary to allow the transaction to complete and be published on Ethereum.
+The transaction will not be added to the blockchain.
+
+For more information about estimating gas, and how this API formulates the transaction costs, see the
+[Estimate transaction costs](../../build-on-linea/gas-fees.mdx) topic.
+
+:::note We recommend using linea_estimateGas for more accurate results
+
+The API uses the same inputs as the standard [eth_estimateGas](https://docs.infura.io/api/networks/linea/json-rpc-methods/eth_estimategas),
+but returns the recommended gas limit, the base fee per gas, and the priority fee per gas.
```

Comment:
```suggestion
:::note

`linea_estimateGas` uses the same inputs as the standard [`eth_estimateGas`](https://docs.infura.io/api/networks/linea/json-rpc-methods/eth_estimategas),
but returns the recommended gas limit, the base fee per gas, and the priority fee per gas. We recommend using `linea_estimateGas` for more accurate results.
```

The note title is hard to read since it's long and all caps.